malboxes.py

```diff
@@ -298,27 +325,20 @@ def reg(parser, args):
         print("Registry modification type invalid.")
         print("Valid ones are: add, delete and modify.")
 
-    filename = os.path.join("scripts","windows", "{}.ps1".format(args.profile))
-    f = open(filename, "a")
-    f.write(line)
-    f.close()
+    filename = os.path.join("scripts", "user",
+                            "windows", "{}.ps1".format(args.profile))
+    append_to_script(filename, line)
 
-    """ Add the script to the profile."""
-    config = load_config(args.profile)
-    provisioners_list = config["provisioners"][0]["scripts"]
-    """ If the script is not already in the profile."""
-    if filename not in provisioners_list:
-        provisioners_list.append(fiString)
```

Comment:
This has never worked. fiString has no other reference. You are not supposed to remove bugs during refactoring :laughing: